lemminx-maven/src/main/java/org/eclipse/lemminx/maven/searcher/RemoteRepositoryIndexSearcher.java

```diff
@@ -214,6 +218,12 @@ public void transferCompleted(TransferEvent transferEvent) {
 		if (context == null) {
 			return CompletableFuture.runAsync(() -> { throw new IllegalArgumentException("context mustn't be null"); });
 		}
+		if ((context.getId().equals("https://repo.maven.apache.org/maven2") || context.getId().equals("central"))
```

Comment:
We could also check eqality for the `CENTRAL_REPO` instead of doing `context.getId().equals("central")`